### DISCUSSION TITLE
How long does it take for my PR request to be approved?

### DISCUSSION THREAD

**Discussion by sungminkim01** [community/community#23544] [Pull Requests]
<p>Hi guys!</p>
<p>I am planning to create my own custom Tokens on Solana and add it to the list so that my tokens may appear with the icon in my wallet.</p>
<p>I wanted to know how long does it take for my PR request to be approved so that my token will be reflected in the wallet?</p>
<p>Thank you!</p>

**Comment by jsoref** [comment, marked as answer]
<p>That’s outside the scope of this forum (which is generic help with GitHub).</p>
<p>Assuming you mean this repository:<br>
<a href="https://github.com/solana-labs/solana/pulls?q=is%3Apr+is%3Aclosed" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/solana-labs/solana/pulls?q=is%3Apr+is%3Aclosed</a><br>
You can review the history of PRs to get a sense of how long it takes for a PR to be merged. Note that generally the time to merge a PR varies significantly even within a repository based on many factors (author, size, kind, quality, season, availability of reviewers, reliability of CI, …).</p>
